README.md

```diff
@@ -1,2 +1,258 @@
+
+class Request:
+    authorizer: AttrDict
+    body: Union[str, dict]
+    path: AttrDict
+    query: AttrDict
+    headers: AttrDict
+    identity: AttrDict
+```
+
+- *Note*: the objects from type ```AttrDict``` are objects built by a dict where the key from dict is a attribute of object. For example:
+
+```python
+from serpens.api import AttrDict
+
+obj = AttrDict({"foo": "bar"})
+obj.foo # bar
+```
+
+## Schema
+- The Schema is a base class for create new classes with follow features:
+> - Static type check
+> - Method for convert a object to dict
+> - Method for create a object from json
+> - Method for create object from dict
+> - Method for dumps a object to string
+
+##### Create a schema
+
+```python
+from serpens.schema import Schema
+from dataclasses import dataclass
+
+@dataclass
+class PersonSchema(Schema):
+    name: str
+    age: int
+```
+##### Create a schema object
+
+```python
+person = PersonSchema('Mike', 30)
+
+print(person.name)
+print(person.age)
+```
+
+##### Create a schema object from a dict.
+
+```python
+person_obj = PersonSchema.load({'name': 'Mike', 'age': 18})
+
+print(person_obj.name)
+print(person_obj.age)
+```
+
+##### Create a schema object from a json string.
+
+```python
+import json
+data = json.dumps({'name': 'mike', 'age': 20})
+person_obj = PersonSchema.loads(data)
+
+print(person_obj.name)
+print(person_obj.age)
+```
+
+##### Convert a schema object to dict.
+
+```python
+p1 = PersonSchema('Mike', 30)
+person_dct = PersonSchema.dump(p1)
+
+print(person_dct['name'])
+print(person_dct['age'])
+```
+
+##### Convert a schema object to json string.
+
+```python
+p1 = PersonSchema('Mike', 30)
+person_str = PersonSchema.dumps(p1)
+
+print(person_str)
+```
+
+## CSV Utils
+
+- Utility for read and write csv. This utility is useful for read csv with BOM or read csv encoded as ISO-8859. 
+
+##### Read CSV
+
+```python
+from serpens import csvutils as csv
+
+dict_reader = csv.open_csv_reader('fruits_iso8859.csv')
+
+for record in dict_reader:
+    print(record)
+```
+
+##### Write CSV
+
+```python
+from serpens import csvutils as csv
+
+writer = csv.open_csv_writer('out.csv')
+writer.writerow(["id", "name"])
+writer.writerow(["1", "Açaí"])
+
+del writer
+```
+
+## Database
+
+TODO: 
+This utilities are useful for works with database.
+
+##### Migrate databases
+
+- This migrations use yoyo-migration.
+
+```python
+from serpens import database
+
+database_url = "postgres://user:password@host/db"
+path = "/path/to/migrations" # yoyo migrations
+
+database.migrate(database_url, path)
+```
+
+##### Create a Pony Database instance
+
+"*The Database object manages database connections using a connection pool.*"
+
+```python
+from serpens import database
+
+database_url = "postgres://user:password@host/db"
+db = database.setup(database_url)
+print(db.provider_name)
+```
+
+## DynamoDB Documents
+
+The serpens provide a base class (called *BaseDocument*) for work with tables from DynamoDB. 
```

Comment:
aqui também caberia "for working" ou "to work".

No começo acho que fica melhor "Serpens provides...."